mflowgen/glb_top/construct.py

```diff
@@ -35,15 +35,15 @@ def construct():
     'flatten_effort' : 3,
     'topographical'  : True,
     # hold target slack
-    'hold_target_slack'   : 0.03,
+    'hold_target_slack'   : 0.04,
     # array_width = width of CGRA below GLB; `pin-assignments.tcl` uses
     # these parms to set up per-cgra-column ports connecting glb tile
     # signals in glb_top to corresponding CGRA tile columns below glb_top
     'array_width'         : 32,
     'num_glb_tiles'       : 16,
     'tool'                : "VCS",
     # glb tile memory size (unit: KB)
-    'use_container' : True,
+    'use_container' : False,
```

Comment:
I'd prefer to leave this as true so we know that we're always using the lastest working version of aha to generate rtl.

Reply:
okay. I will change it to `True`.